src/main/java/moneytransferapp/MoneyTransferWorkflowImpl.java

```diff
@@ -0,0 +1,34 @@
+package moneytransferapp;
+
+import io.temporal.activity.ActivityOptions;
+import io.temporal.workflow.Workflow;
+ import io.temporal.common.RetryOptions;
+import java.time.Duration;
+
+// @@@SNIPSTART money-transfer-project-template-java-workflow-implementation
+public class MoneyTransferWorkflowImpl implements MoneyTransferWorkflow {
+
+    RetryOptions retryoptions =  RetryOptions.newBuilder()
+            .setInitialInterval(Duration.ofSeconds(1))
+            .setMaximumInterval(Duration.ofSeconds(100))
+            .setBackoffCoefficient(2)
+            .setMaximumAttempts(500)
+            .build();
+
+    private final ActivityOptions options = ActivityOptions.newBuilder()
+            .setStartToCloseTimeout(Duration.ofSeconds(5))
+            .setRetryOptions(retryoptions)
```

Comment:
These retry options are already provided by default.

Reply:
We are trying to decide which has more weight with a new user - retrying by default without specifying a policy and mention that they are customizable, or showing how easy it is to specify options and then mention that you don't need to because Temporal does it by default.